Cargo.toml

```diff
@@ -11,15 +11,12 @@ bs58 = {version = "0.4", features=["check"]}
 signature = "*"
 rand_core = { version = "0.6", features = ["getrandom"] }
 ed25519-dalek = { git = "https://github.com/helium/ed25519-dalek", branch = "madninja/bump_rand" }
-p256 = { git = "https://github.com/helium/elliptic-curves", branch="rg/compact", default-features=false, features=["arithmetic", "ecdsa", "sha256", "zeroize"] }
+p256 = { git = "https://github.com/helium/elliptic-curves", branch="madninja/compact_point_impl", default-features=false, features=["arithmetic", "ecdsa", "sha256", "zeroize"] }
```

Comment:
May want to drag the elliptic-curves branch and sync it upstream before landing this one, your call.